scheduler/planner.go

```diff
@@ -227,6 +227,11 @@ func (unit *Unit) RankValue() int64 {
 		// would also be scheduled in a version.
 		priority += length
 	}
+	if generateTask {
+		// give generators a boost so people don't have to wait twice.
+		priority += 100
+		unit.cachedValue += priority * unit.distro.GetGenerateTaskFactor()
```

Comment:
I don't totally follow. I would expect that adding weighted priority would happen here, but actually adding it to unit.cachedValue would happen elsewhere, or else we would re-add several times.

Reply:
good point, this is an oversight. I'll just bump priority.